org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/StatusBar.java

```diff
@@ -173,11 +175,15 @@ public void setStatusLine (String text, boolean error, ProcessInfoLog[] m_logs)
 			return;
 		
     	String labelText = buildLabelText(m_statusText);
+    	//iDempiereConsulting __05/10/2021 --- Error Message - The popup exposes the complete message and life time can be configured through system variable configurator (ErrorMsgLifeTime)  
+    	// If the value equals '0', then the popup will not disappear and you will need to close it manually; otherwise set the variable as milleseconds
+    	int duration = Integer.parseInt(MSysConfig.getValue("ErrorMsgLifeTime", "4500", Env.getAD_Client_ID(Env.getCtx())));   //MSysConfig.getIntValue("ErrorMsgLifeTime", 3500);
```

Comment:
for zk sysconfig, the common convention is to use all cap, separate by _ and starts with ZK_. Also, add the constant to MSysConfig (note existing constant is alphabetically order) instead of using the string literal here.

Reply:
And please remove the "//MSysConfig.getIntValue("ErrorMsgLifeTime", 3500);" comment.

Reply:
And you need to create a tenant level, default ad_sysconfig record (with official id) with value equal to the current hard coded duration.